nsls2forge_utils/dashboard.py

```diff
@@ -5,6 +5,35 @@
 functions had to be re-implemented here.
 """
 from .all_feedstocks import get_all_feedstocks
+from .meta_utils import get_attribute
+
+
+def _extract_org(pkg):
```

Comment:
I would recommend to call this function so that we understand that we are trying to extract the GitHub org here.